### PR TITLE
split create and configure into seperate playbooks, implemented teardown

### PR DESCRIPTION
**Pull request by NatalieClaraPetersen**

### CODE REVIEW

File: ansible/create_droplets.yml [hunk 1]
```diff
@@ -42,97 +45,28 @@
...
         workers: "{{ workers_created.results | map(attribute='data.droplet') | list }}"
 
-- name: Configure all Droplets
-  hosts: all
-  tasks:
-    - name: Update apt cache
-      apt:
-        update_cache: yes
-        cache_valid_time: 3600   # Set cache validity time to one hour
+# TEARDOWN (only run when the playbook is run with: --tags teardown)
```
Comment (rlnprogrammering) [inline ansible/create_droplets.yml:48]: This is cool!

File: ansible/create_droplets.yml [hunk 2]
```diff
@@ -42,97 +45,28 @@
...
-
-    - set_fact:
-        SWARM_JOIN_TOKEN: "{{ swarm_join_token.stdout }}"
-        SWARM_MANAGER_IP: "{{ ansible_host }}"
-
+        unique_name: true
+        api_token: "{{ DIGITAL_OCEAN_TOKEN }}"
+      loop: "{{ workers_created.results | map(attribute='data.droplet.name') | list }}"
+      when: workers_created is defined
+      tags: teardown,never
```
Comment (rlnprogrammering) [inline ansible/create_droplets.yml:66]: What does the `never` tag do here?

**Review by rlnprogrammering — APPROVED** [review]
Looks cool, i assume the `never` tag specifies it should only run when the other tag `teardown` is used?

File: ansible/create_droplets.yml (hunk 2 above)
Comment (NatalieClaraPetersen) [reply ansible/create_droplets.yml:66]: that it is only run when specifying "--tags teardown" when running the playbook

Comment (NatalieClaraPetersen) [reply ansible/create_droplets.yml:66]: Ah yes, so you guessed correctly

**Pull request merged**